### DISCUSSION TITLE
B站已经开始进行小范围的新界面的内测

### DISCUSSION THREAD

**Discussion by SekiBetu** [the1812/Bilibili-Evolved#2209] [综合讨论 / General]
![image](https://user-images.githubusercontent.com/38283893/130633348-a8d7b291-9952-48e1-b2f2-cdfa9a07540b.png)
只是提一句，等大规模部署了再说，目前只有首页的暗黑模式失效，别的页面没有变化

**Comment by the1812** [comment]
感觉比上一版好看些了

**Comment by SekiBetu** [comment]
![image](https://user-images.githubusercontent.com/38283893/130634389-9c6a43ef-9984-4cdd-ad20-5a4634cc8b6f.png)
![image](https://user-images.githubusercontent.com/38283893/130634416-0ea1e611-bcdf-4cef-ab55-9667d3e336fa.png)
![image](https://user-images.githubusercontent.com/38283893/130634447-f0fef207-528e-4375-af7a-dbbdb9bc3e56.png)
![image](https://user-images.githubusercontent.com/38283893/130634459-9a5439a9-bc3b-4b3a-b1d7-6fb855e41e4d.png)

关了脚本是这样的

**Reply by SekiBetu** [reply]
![image](https://user-images.githubusercontent.com/38283893/130634639-9444f37e-c412-447a-ba65-8557b789c3e0.png)
![image](https://user-images.githubusercontent.com/38283893/130634665-d117cd08-6261-43fe-9bf1-478c401eefde.png)
![image](https://user-images.githubusercontent.com/38283893/130634680-654fe91b-be2e-44e0-9355-b0e07f1d4732.png)
![image](https://user-images.githubusercontent.com/38283893/130634714-6684a086-d601-46f0-9083-64aa937b6c77.png)
![image](https://user-images.githubusercontent.com/38283893/130634745-591edfa7-6e73-439b-9343-d40227c460da.png)
![image](https://user-images.githubusercontent.com/38283893/130634793-63071b7a-0039-4c98-b167-572433358c7a.png)
![image](https://user-images.githubusercontent.com/38283893/130634816-5120f4aa-d1af-4748-9b9d-7f22ac2ac9e0.png)

**Reply by kethoven** [reply]
播放界面播放器尺寸改了吗？

**Comment by u2shana** [comment]
不把稍后再看直接外放出来，我还是不会用原生的

**Reply by kethoven** [reply]
播放器界面改版了吗？

**Reply by the1812** [reply]
没有吧, 这个只是首页的

**Reply by kethoven** [reply]
希望播放器的尺寸能够增大一些，看惯了其他的视频平台，感觉b站的播放器偏小了。

**Comment by kethoven** [comment]
希望播放器界面变大一些，看惯了其他视频平台，感觉b站的播放器有点小了。

**Comment by PRSJMC** [comment]
同样遇到了首页改版，导致简化首页“失效”的问题，实际上是被挤到最下面了，如下：
<img width="900" alt="Snipaste_2021-09-08_17-40-01" src="https://user-images.githubusercontent.com/46528990/132486113-7c761db3-658d-4912-b6b6-107816b34002.png">

**Comment by the1812** [comment]
<img src="https://user-images.githubusercontent.com/26504152/132937618-f870ada2-c6b4-4f02-98b0-893fae131625.png" width="50">新版首页这图片比例... 

![image](https://user-images.githubusercontent.com/26504152/132937578-a87fd0ae-4f00-4464-8301-eda04c68cd99.png)

**Comment by DoraTiger** [comment]
可以加个判断，检测到新界面并开始简化首页的时候，直接请求旧版首页吗。今天突然灰度到我了，两台电脑一台新主页一台旧主页。旧主页那台电脑，edge是旧的，Firefox是新的，就离谱。

**Reply by the1812** [reply]
右下角有返回旧版

**Reply by DoraTiger** [reply]
嗯，我果断切回去了，还是简洁主页舒服。

**Comment by u2shana** [comment]
伴随这次改版,最近网页版首页推荐不知道怎么的全是无相关推荐,app端还是正常的推荐内容

**Comment by SekiBetu** [comment]
我好像被踢出内测了，主页变成老版本了

**Reply by the1812** [reply]
https://www.bilitools.top/read/1

**Reply by the1812** [reply]
糟了我也被踢了, 改 cookie 也不行了

**Comment by FoundTheWOUT** [comment]
我现在只要不登录都是新版
